src/SFML/Window/Win32/InputImpl.hpp

```diff
@@ -31,6 +31,7 @@
 #include <SFML/Window/Keyboard.hpp>
 #include <SFML/Window/Mouse.hpp>
 
+#include <windows.h>
```

Comment:
Is this needed to get access to `WORD`?

Reply:
There's no need for it to be declared in the header, so I've removed it and the include now